src/test/java/org/octri/messaging/autoconfig/MessagingConfigTest.java

```diff
@@ -0,0 +1,42 @@
+package org.octri.messaging.autoconfig;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.system.CapturedOutput;
+import org.springframework.boot.test.system.OutputCaptureExtension;
+
+@ExtendWith(OutputCaptureExtension.class)
```

Comment:
I didn't realize that Spring Boot provided this. It captures all output including log messages, which can be used to verify that the expected warning is logged.